pycon_archive_past_website/pycon_archive_past_website.py

```diff
@@ -317,16 +79,18 @@ def main():
     required=True,
 )
 @click.option(
-    "--base", "base_url", help="The base url for all site", type=str, required=False
+    "--base",
+    "base_url",
+    help="The base url for all site",
+    type=str,
+    required=False,
+    default="",
 )
 def check_year(param, base_url):
     """Get Pycon Website According To the Year"""
-    global PYCON_YEAR
-    global BASE_URL  # TODO: [Refactor] to encapuslate all the global variable
-    BASE_URL = base_url
-    PYCON_YEAR = str(param.year)
-    if PYCON_YEAR >= "2016" and PYCON_YEAR <= "2020":
-        main()
+    year: str = str(param.year)
+    if year >= "2016" and year <= "2020":
```

Comment:
I think we can remove the specific range of website year. For future, we just need to update the customize crawler in website folder.

Reply:
Thanks for the feedback, I'll update this part.

Reply:
Updated.